lookout/style/format/rules.py

```diff
@@ -419,11 +430,75 @@ def _merge_rules(cls, rules: List[Rule]) -> List[Rule]:
             new_rules.append(Rule(tuple(new_rule), stats))
         return new_rules
 
+    @staticmethod
+    def _get_parents(tree: Tree.Tree) -> Dict[int, int]:
```

Comment:
You use `_get_parents` only once and the function is small, better to inline it.